.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,7 +19,7 @@ Also go through our FAQ: https://github.com/iTwin/iTwinUI/wiki/FAQ
 <!-- A clear and concise description of what you expected to happen. -->
 
 #### Reproduction
-Link to a minimal repro: <!-- fork this sandbox: https://codesandbox.io/s/itwinui-react-example-son74 -->
+Link to a minimal repro: <!-- fork this sandbox: https://codesandbox.io/s/github/iTwin/iTwinUI/tree/main/playgrounds/cra -->
```

Comment:
The link to new sandbox will not work until this PR is merged. In the meantime, use this branch url: https://codesandbox.io/s/github/iTwin/iTwinUI/tree/mayank/fix-instructions/playgrounds/cra